webdriver/tests/bidi/browsing_context/print/margin.py

```diff
@@ -105,14 +109,55 @@ async def test_margin_same_as_page_dimension(
     await bidi_session.browsing_context.navigate(
         context=top_context["context"], url=page, wait="complete"
     )
-    print_value = await bidi_session.browsing_context.print(
+
+    # This yields and empty content area: https://github.com/w3c/webdriver-bidi/issues/473
```

Comment:
```suggestion
    # This yields an empty content area: https://github.com/w3c/webdriver-bidi/issues/473
```

Probably we want to move this test to `invalid.py`